src/main/scala/com/github/xiaodongw/swagger/finatra/SwaggerController.scala

```diff
@@ -6,17 +6,21 @@ import com.twitter.finatra.response.Mustache
 import io.swagger.models.Swagger
 import io.swagger.util.Json
 
-@Mustache("index")
-case class SwaggerView(title: String, path: String)
-
 class SwaggerController(docPath: String = "/api-docs", swagger: Swagger) extends Controller {
+
   get(s"${docPath}/model") { request: Request =>
     response.ok.body(Json.mapper.writeValueAsString(swagger))
       .contentType("application/json").toFuture
   }
 
   get(s"${docPath}/ui") { request: Request =>
-    response.temporaryRedirect
-        .location("/webjars/swagger-ui/2.2.8/index.html?url=/api-docs/model")
+    redirectToSwaggerUi()
   }
+
+  get(s"${docPath}") { request: Request =>
```

Comment:
True, I'll remove it .

Can we do this change in another commit ? 
The endpoint docPath shouldn't be used for something else and in my opinion it makes sense to reply with the swagger ui when only docPath is requested (which is shorter then docPath/ui)